.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
 npm-debug.log
 .DS_Store
-.idea
\ No newline at end of file
+.idea
+.vscode
```

Comment:
Cool. If btw you have ended up setting up custom Visual Studio code styles setting for d3-x3dom please do feel free to add the settings config (if there is one) in the /config/ folder.
Also see notes on https://github.com/jamesleesaunders/d3-x3dom/blob/master/CONTRIBUTING.md#code-styles